vespa-dependencies-enforcer/allowed-maven-dependencies.txt

```diff
@@ -128,9 +128,11 @@ org.apache.opennlp:opennlp-tools:${opennlp.vespa.version}
 org.apache.velocity:velocity-engine-core:${velocity.vespa.version}
 org.apache.yetus:audience-annotations:0.12.0
 org.apache.zookeeper:zookeeper-jute:${zookeeper.client.vespa.version}
+org.apache.zookeeper:zookeeper-jute:3.8.0
 org.apache.zookeeper:zookeeper-jute:3.8.1
 org.apache.zookeeper:zookeeper-jute:3.9.1
 org.apache.zookeeper:zookeeper:${zookeeper.client.vespa.version}
+org.apache.zookeeper:zookeeper:3.8.0
```

Comment:
This looks unexpected. Looks like we still have 3.8.0 being included somewhere else.

Reply:
`vespa-zookeeper-server` uses that version. All those submodules end up in here.

Reply:
3.8.1 is not used anywhere at all, except for in the 3.8.1 module, as far as I can see? And I intend to remove those modules when 3.9.1. is completely rolled out.

Reply:
The `zookeeper.version` property should be used then.